fele-client-service/utils/helpers.js

```diff
@@ -0,0 +1,9 @@
+const { createHash } = require('crypto')
+
+const sha256 = (content) => {  
+    return createHash('sha256').update(content).digest('hex')
```

Comment:
Don't we need a secret string?

Reply:
As of now, we don't have a secret string to minimize complexity and keep it to simple comparison of hashed passwords. We just store the hashed value in database, pull it and compare it with the hashed value of the input password that the use enters. What do you think? We can have a discussion on this.